packages/components/src/components/progress-bar-onboarding/progress-bar-onboarding.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
-
 import './progress-bar-onboarding.scss';
 
 type TProgressBarOnboarding = {
     step: number;
     amount_of_steps: Array<string>;
     setStep: React.Dispatch<React.SetStateAction<number>>;
+    is_transition?: boolean;
 };
 
-const ProgressBarOnboarding = ({ step, amount_of_steps, setStep }: TProgressBarOnboarding) => {
+const ProgressBarOnboarding = ({ step, amount_of_steps, setStep, is_transition }: TProgressBarOnboarding) => {
```

Comment:
@farzin-deriv In my opinion, because of `is_transition` is optional so we don't need to add default value, because if we don't pass this value it will be `undefined` by default what is equal to `false`. Let me know if I'm wrong.

Reply:
@sergei-deriv Yes you are right, The reason for giving a default value to optional booleans is just to have better types, So `is_transition` instead of being `boolean | undefined` will just be `boolean`.